site/content/docs/manifest/lb-web-service.md

```diff
@@ -199,8 +208,34 @@ count:
   response_time: 2s
 ```
 
-<span class="parent-field">count.</span><a id="count-range" href="#count-range" class="field">`range`</a> <span class="type">String</span>  
-Specify a minimum and maximum bound for the number of tasks your service should maintain.  
+<span class="parent-field">count.</span><a id="count-range" href="#count-range" class="field">`range`</a> <span class="type">String or Map</span>
+You can specify a minimum and maximum bound for the number of tasks your service should maintain.
+```yaml
+count:
+  range: n-m
+```
+This will set up an Application Autoscaling Target with the `MinCapacity` of `n` and `MaxCapacity` of `m`.
+
+Alternatively, if you wish to scale your service onto Fargate Spot instances, specify `min` and `max` under `range` and then specify `spot_from` with the desired count you wish to start placing your services onto spot capacity. For example:
+
+```yaml
+count:
+  range:
+    min: 1
+    max: 10
+    spot_from: 3
+```
+
+This will set your range as 1-10 as before, but will place two copies of your service on dedicated Fargate capacity. Once your service scales to 3 copies, they will be placed on spot until the maximum is reached.
```

Comment:
```suggestion
This will set your range as 1-10 as before, but will place the first two copies of your service on dedicated Fargate capacity. If your service scales to 3 copies, the third and any additional copies will be placed on spot until the maximum is reached.
```